examples/atmajs/index.html

```diff
@@ -11,34 +11,38 @@
 
 		TodoMVC Atma.js Application
 
-		1. Read readme.md - you will get basic information about the libraries
-		2. Hint: Viewing *.mask files enable javascript or less syntax highlighting in your IDE
+		1. Read the readme.md - you will get basic information about the libraries
+		2. Hint: Use Sublime/Atom plugins to highlight the *.mask syntax,
```

Comment:
> Use Sublime/Atom plugins to highlight the *.mask syntax,

Seems like more somthing for the readme than for comments here. In fact I think this whole comment block would fit better in the readme.